pom.xml

```diff
@@ -172,6 +184,35 @@
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-slf4j-impl</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
```

Comment:
these 2 dependenices are exist in folio-kafka-wrapper: kafka-clients, kafka-junit
If possible we can use them from this shared library

Reply:
Both are necessary as kafka-junit in needed to startup embedded cluster, and without kafka-clients the tests fail to work properly with kafka